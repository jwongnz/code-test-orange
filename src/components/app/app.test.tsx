import React from 'react';
import App from './app';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('App', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <App />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});