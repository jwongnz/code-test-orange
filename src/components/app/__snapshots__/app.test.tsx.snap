// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<styled.div>
  <BrowserRouter>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(withRouter(CustomerAddOrUpdate))",
          "type": [Function],
        }
      }
      path="/saveCustomer/:id?"
    />
  </BrowserRouter>
</styled.div>
`;
